### ISSUE TITLE
replace deprecating <licenseUrl> from nuget package

### ISSUE THREAD

**Issue by genlu** [dotnet/roslyn-analyzers#2220]
Getting this warning when publishing nuget packages

> WARNING: <licenseUrl> element will be deprecated,please consider switching to specifying the license in the package. Learn more: https://aka.ms/deprecateLicenseUrl ,This package will only be available to download with SemVer 2.0.0 compatible NuGet clients,such as Visual Studio 2017 (version 15.3) and above or NuGet client 4.3 and above. For more information,see https://go.microsoft.com/fwlink/?linkid=852248.

**Comment by mavasani** [comment]
Isn't this something that @tmat was planning to address in arcade SDK itself? Or this needs a follow-up change in the repo?

**Comment by genlu** [comment]
Might be, I don't know how it's handled in the repo. Please close this if that's the case. Thanks!

**Comment by mavasani** [comment]
@genlu I am not sure how this will be fixed (in the arcade repo or needs a fix here). I'd let Tomas give us more info before closing this out.

**Comment by tmat** [comment]
Just did it for Roslyn: https://github.com/dotnet/roslyn/pull/34172

**Comment by tmat** [comment]
Note these lines in the PR:
```
<Copyright>$(CopyrightMicrosoft)</Copyright>
<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>	    
```
